pkg/controller/garbagecollector/dump.go

```diff
@@ -106,106 +120,120 @@ namespace=%v
 		label = label + conditionString + "\n"
 	}
 
-	return []encoding.Attribute{
-		{Key: "label", Value: fmt.Sprintf(`"%v"`, label)},
+	return []attribute{
+		{Key: "label", Value: label},
 		// these place metadata in the correct location, but don't conform to any normal attribute for rendering
-		{Key: "group", Value: fmt.Sprintf(`"%v"`, v.gvk.Group)},
-		{Key: "version", Value: fmt.Sprintf(`"%v"`, v.gvk.Version)},
-		{Key: "kind", Value: fmt.Sprintf(`"%v"`, v.gvk.Kind)},
-		{Key: "namespace", Value: fmt.Sprintf(`"%v"`, v.namespace)},
-		{Key: "name", Value: fmt.Sprintf(`"%v"`, v.name)},
-		{Key: "uid", Value: fmt.Sprintf(`"%v"`, v.uid)},
-		{Key: "missing", Value: fmt.Sprintf(`"%v"`, v.missingFromGraph)},
-		{Key: "beingDeleted", Value: fmt.Sprintf(`"%v"`, v.beingDeleted)},
-		{Key: "deletingDependents", Value: fmt.Sprintf(`"%v"`, v.deletingDependents)},
-		{Key: "virtual", Value: fmt.Sprintf(`"%v"`, v.virtual)},
+		{Key: "group", Value: v.gvk.Group},
+		{Key: "version", Value: v.gvk.Version},
+		{Key: "kind", Value: v.gvk.Kind},
+		{Key: "namespace", Value: v.namespace},
+		{Key: "name", Value: v.name},
+		{Key: "uid", Value: string(v.uid)},
+		{Key: "missing", Value: fmt.Sprintf(`%v`, v.missingFromGraph)},
+		{Key: "beingDeleted", Value: fmt.Sprintf(`%v`, v.beingDeleted)},
+		{Key: "deletingDependents", Value: fmt.Sprintf(`%v`, v.deletingDependents)},
+		{Key: "virtual", Value: fmt.Sprintf(`%v`, v.virtual)},
 	}
 }
 
-// NewGonumVertex creates a new gonumVertex.
-func NewGonumVertex(node *node, nodeID int64) *gonumVertex {
+// NewdotVertex creates a new dotVertex.
+func NewdotVertex(node *node) *dotVertex {
```

Comment:
`NewDOTVertex` to match other abbreviations.

Reply:
done